demo/KevinDemo/Stories/Payment/PaymentView.swift

```diff
@@ -0,0 +1,186 @@
+//
+//  PaymentView.swift
+//  KevinDemo
+//
+//  Created by Edgar Žigis on 8/1/21.
+//  Copyright © 2021 kevin.. All rights reserved.
+//
+
+import Localize_Swift
+import Kevin
+import SwiftUI
+import UIKit
+import HalfASheet
+
+struct PaymentView: View {
+    
+    @StateObject var viewModel = PaymentViewModel()
+    
+    var body: some View {
+        NavigationView {
+            ZStack {
+                Color("PrimaryBackgroundColor").ignoresSafeArea()
```

Comment:
maybe use background(Color("PrimaryBackgroundColor")) this way code is more consistent and you can remove ZStack